test/fuzz/LendingPool/DoActionWithLeverage.fuzz.t.sol

```diff
@@ -245,10 +247,8 @@ contract DoActionWithLeverage_LendingPool_Fuzz_Test is LendingPool_Fuzz_Test {
         vm.assume(liquidity >= amountLoaned);
         vm.assume(amountLoaned > 0);
 
-        uint256 fee = uint256(amountLoaned) * originationFee / 10_000;
-
         vm.prank(users.creatorAddress);
-        pool.setOriginationFee(originationFee);
```

Comment:
Why fee to 0?

Reply:
To simplify this test which just checks for an event emittal, fees are tested in other tests.